CPAC/pipeline/nipype_pipeline_engine/engine.py

```diff
@@ -134,6 +142,56 @@ def __init__(self, *args, mem_gb=DEFAULT_MEM_GB, **kwargs):
             * 'xyz' (spatial)
             * 't' (temporal)''']))  # noqa E501
 
+    def _add_flags(self, flags):
+        '''
+        Parameters
+        ----------
+        flags : list
```

Comment:
So this is a list of length 2, where the first is a list/string of the flags we want, and the second is what flags were specified before our modificaton?

Reply:
:joy: I forgot to finish this docstring! Your intuition is close to what I wrote: if it's a single list, it's just flags to add. If it's a 2-tuple, `flags[0]` is to add and `flags[1]` is to remove